sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -4,8 +4,10 @@
 
 EAPI=6
 
+: ${CMAKE_MAKEFILE_GENERATOR:=ninja}
```

Comment:
Because it scales better than recursive make. Also, for consistency with other LLVM ebuilds.